### PR TITLE
Release v4.3.0

### PR DESCRIPTION
**Pull request by vcerenu**
This PR bumps version to Wazuh 4.3.0 version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Deploy the Wazuh platform using Chef cookbooks. Chef recipes are prepared for in
 
 | Wazuh version | Elastic | ODFE   |
 |---------------|---------|--------|
+| v4.3.0        | v7.10.2 | v1.13.2|
+|---------------|---------|--------|
```
Comment (myu1d157h0u54nd) [inline README.md]: Remove: |---------------|---------|--------|

**Review by myu1d157h0u54nd — COMMENTED** [review]
LGTM

**Review by okynos — APPROVED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (vcerenu) [reply README.md]: Done

**Review by teddytpc1 — APPROVED** [review]
LGTM

**Pull request merged**